### PR TITLE
Upgrade to 1.7.1

### PR DESCRIPTION
**Pull request by ijunaid8989**

### CODE REVIEW

File: lib/evercam_media/archive_creator/archive_creator.ex [hunk 1]
```diff
@@ -63,7 +62,8 @@ defmodule EvercamMedia.ArchiveCreator.ArchiveCreator do
         end
       rescue
         error ->
-          Util.error_handler(error)
+          Logger.error inspect(error)
```
Comment (azharmalik3) [inline lib/evercam_media/archive_creator/archive_creator.ex]: Why did you add same lines here?

Comment (ijunaid8989) [reply lib/evercam_media/archive_creator/archive_creator.ex]: Due to this warning
```
warning: System.stacktrace/0 outside of rescue/catch clauses is deprecated. If you want to support only Elixir v1.7+, you must access __STACKTRACE__ inside a rescue/catch. If you want to support earlier Elixir versions, move System.stacktrace/0 inside a rescue/catch
  lib/evercam_media/util.ex:103
```
`Util.error_handler(error)` This method was doing a stack trace, according to warning it should be written directly in rescue or catch.
